node/cli/src/command.rs

```diff
@@ -211,16 +211,28 @@ fn extract_genesis_wasm(chain_spec: &Box<dyn sc_service::ChainSpec>) -> Result<V
 		.ok_or_else(|| "Could not find wasm file in genesis state!".into())
 }
 
-/// Parse command line arguments into service configuration.
-pub fn run() -> Result<()> {
-	let mut cli = Cli::from_args();
-
+fn validate_trace_environment(cli: &Cli) -> Result<()> {
 	if (cli.run.ethapi.contains(&EthApi::Debug) || cli.run.ethapi.contains(&EthApi::Trace))
-		&& cfg!(not(feature = "evm-tracing"))
+		&& cli
+			.run
+			.base
+			.base
+			.import_params
+			.wasm_runtime_overrides
+			.is_none()
 	{
-		return Err("`--ethapi` only available on `evm-tracing` feature.".into());
+		return Err(
+			"`debug` or `trace` namespaces requires `--wasm-runtime-overrides /path/to/overrides`."
```

Comment:
This error will bother us in the case of a native execution, we should provide an exception for this case. This would avoid adding `--wasm-runtime-overrides` in the CI for nothing.

Reply:
Same